### PR TITLE
Ready for production #1

### PR DESCRIPTION
**Pull request by rakshi00**
ready for tests with few volunteers

### CODE REVIEW

File: backend/src/main/java/de/njsm/versusvirus/backend/service/customer/CustomerDTO.java [hunk 1]
```diff
@@ -16,13 +16,24 @@
...
-        firstName = customer.getFirstName();
-        lastName = customer.getLastName();
-        phone = customer.getPhone();
-        mobile = customer.getMobile();
-        address = customer.getAddress().getAddress();
-        zipCode = customer.getAddress().getZipCode();
-        city = customer.getAddress().getCity();
+        if (customer.isDeleted()) {
+            uuid = customer.getUuid();
+            firstName = "gelöscht";
```
Comment (F1rst-Unicorn) [inline backend/src/main/java/de/njsm/versusvirus/backend/service/customer/CustomerDTO.java:21]: Created issue #75 for inconsistent wording

**Review by F1rst-Unicorn — APPROVED** [review]

File: backend/src/main/java/de/njsm/versusvirus/backend/service/customer/CustomerDTO.java (hunk 1 above)
Comment (F1rst-Unicorn) [reply backend/src/main/java/de/njsm/versusvirus/backend/service/customer/CustomerDTO.java:21]: maybe I just didn't check internal vs client-facing strings, not sure

**Comment by ntruessel** [comment]
Testing now contains password change functionality, do we want to update or do we release this the next time?

**Comment by ntruessel** [comment]
Before Releasing we should clean up the DB

**Comment by rakshi00** [comment]
> Before Releasing we should clean up the DB

Agreed.

Maybe the wording was not good enough. Next steps look as follows:
1. Testing in project group Elsa hilft
2. Testing in volunteer group with real purchases [DB clean-up has to be done before this stage]
3. Roll-out to all volunteers, real go-live

So I for not cleaning DB now.

password change should be included asap.

**Pull request merged**